pkg/task/common.go

```diff
@@ -78,9 +78,14 @@ func DefineCommonFlags(flags *pflag.FlagSet) {
 	flags.String(flagKey, "", "Private key path for TLS connection")
 
 	flags.Uint64(flagRateLimit, 0, "The rate limit of the task, MB/s per node")
-	flags.Uint32(flagConcurrency, 4, "The size of thread pool on each node that executes the task")
 	flags.Bool(flagChecksum, true, "Run checksum at end of task")
 
+	// Default concurrency is different for backup and restore.
+	// Leave it 0 and let them adjust the value.
+	flags.Uint32(flagConcurrency, 0, "The size of thread pool on each node that executes the task")
+	// It may confuse users , so just hide it.
+	_ = flags.MarkHidden(flagConcurrency)
```

Comment:
🤔 but i don't find this confusing

Reply:
`--help` prints "concurrency .... (default 0)", I find this is confusing. :joy:

Reply:
`(default 0)` should not appear, otherwise the `--rate-limit` would also show `(default 0)`.